test/corpus/expressions.txt

```diff
@@ -627,6 +627,22 @@ R"(a)" R"(b)" R"(c)";
   (expression_statement (concatenated_string (raw_string_literal) (raw_string_literal) (raw_string_literal)))
   (expression_statement (concatenated_string (string_literal) (raw_string_literal) (string_literal) (raw_string_literal))))
 
+============================================
+User-defined literals
+============================================
+
+1.2_km;
+"foo" "bar"_baz;
+---
+
+(translation_unit
+  (expression_statement
+    (user_defined_literal
+      literal: (number_literal)))
+  (expression_statement
+    (user_defined_literal
+      literal: (concatenated_string (string_literal) (string_literal)))))
```

Comment:
You specified `suffix` as a named field but I don't see it in these tests.  Can you put the `suffix` field descriptor here so both `literal:` and `suffix:` are shown?

Reply:
As mentioned in my other reply, since it doesn't seem to work with `token.immediate()`, would it be better if I removed the `liternal` named field too to be consistent?